spec/schemas/_common.aggregations.yaml

```diff
@@ -1450,152 +1452,367 @@ components:
-            moving_avg:
-              $ref: '#/components/schemas/MovingAverageAggregation'
-            moving_percentiles:
-              $ref: '#/components/schemas/MovingPercentilesAggregation'
-            moving_fn:
-              $ref: '#/components/schemas/MovingFunctionAggregation'
-            multi_terms:
-              $ref: '#/components/schemas/MultiTermsAggregation'
-            nested:
-              $ref: '#/components/schemas/NestedAggregation'
-            normalize:
-              $ref: '#/components/schemas/NormalizeAggregation'
-            parent:
-              $ref: '#/components/schemas/ParentAggregation'
-            percentile_ranks:
-              $ref: '#/components/schemas/PercentileRanksAggregation'
-            percentiles:
-              $ref: '#/components/schemas/PercentilesAggregation'
-            percentiles_bucket:
-              $ref: '#/components/schemas/PercentilesBucketAggregation'
-            range:
-              $ref: '#/components/schemas/RangeAggregation'
-            rare_terms:
-              $ref: '#/components/schemas/RareTermsAggregation'
-            rate:
-              $ref: '#/components/schemas/RateAggregation'
-            reverse_nested:
-              $ref: '#/components/schemas/ReverseNestedAggregation'
-            sampler:
-              $ref: '#/components/schemas/SamplerAggregation'
-            scripted_metric:
-              $ref: '#/components/schemas/ScriptedMetricAggregation'
-            serial_diff:
-              $ref: '#/components/schemas/SerialDifferencingAggregation'
-            significant_terms:
-              $ref: '#/components/schemas/SignificantTermsAggregation'
-            significant_text:
-              $ref: '#/components/schemas/SignificantTextAggregation'
-            stats:
-              $ref: '#/components/schemas/StatsAggregation'
-            stats_bucket:
-              $ref: '#/components/schemas/StatsBucketAggregation'
-            string_stats:
-              $ref: '#/components/schemas/StringStatsAggregation'
-            sum:
-              $ref: '#/components/schemas/SumAggregation'
-            sum_bucket:
-              $ref: '#/components/schemas/SumBucketAggregation'
-            terms:
-              $ref: '#/components/schemas/TermsAggregation'
-            top_hits:
-              $ref: '#/components/schemas/TopHitsAggregation'
-            t_test:
-              $ref: '#/components/schemas/TTestAggregation'
-            top_metrics:
-              $ref: '#/components/schemas/TopMetricsAggregation'
-            value_count:
-              $ref: '#/components/schemas/ValueCountAggregation'
-            weighted_avg:
-              $ref: '#/components/schemas/WeightedAverageAggregation'
-            variable_width_histogram:
-              $ref: '#/components/schemas/VariableWidthHistogramAggregation'
-          minProperties: 1
-          maxProperties: 1
+          oneOf: 
+            - properties: 
```

Comment:
A little late to the party but I don't think this change was necessary and it makes the schema more complex.
`minProperties: 1` and `maxProperties: 1` already enforce it must have exactly 1 aggregation.

`unevaluatedProperties` is also meant for compound `allOf`. For simple objects, `additionalProperties` is the one to use.